.github/workflows/test.yml

```diff
@@ -25,30 +26,9 @@ jobs:
       - name: Setup Scala
         uses: olafurpg/setup-scala@v10
         with:
-          java-version: adopt@1.8
```

Comment:
I merged the two checks into one by making the Java version part of the test matrix.